.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+# Summary
+## Closes
+Some issue
+
+## Changes
+Ipsum lorem.
+ 
+## Specific Testing (Optional)
+Any specific text used to test outside of typical Ruff workflows
```

Comment:
The examples should be commented to avoid having to remove them for each PR.
```suggestion
<!-- Some issue -->

## Changes
<!-- Ipsum lorem. -->
 
## Specific Testing (Optional)
<!-- Any specific text used to test outside of typical Ruff workflows -->
```

Reply:
Thank you!